app/views/maps/_sidebar_exports.html.erb

```diff
@@ -81,6 +111,35 @@
   </div>
 </div>
 <script>
+var exp1 = 0;
+// //var arr = [];
+// var b = $('.export-warpable').length;
+//     console.log(b);
+//     var k = [0, 0, 0];
+//     console.log(k);
+$('.export-warpable').click(function() {
+    var warpable_id = $(this).children('img').attr("data-warpable-id");
+    console.log(warpable_id);
+    if(exp1==0){
+      $(this).css('border', "solid 2px red");
+      $(this).parent().addClass('export-image');
+      exp1 = 1;
+    }
+    else{
+      $(this).css('border', "none");
+      $(this).parent().removeClass('export-image');
+      exp1 = 0;
+    }
+});
+function check_export(){
+  if($('.export-image').length > 0){
+    $('#begin-export').attr('disabled',false).attr('title','Start Export');
+  }
+  else{
+    $('#begin-export').attr('disabled',true).attr('title','Select images to export');
+  }
+};
+
```

Comment:
This function is not being called anywhere, so doesn't work. Also `disabled` would be a `prop` and to change the text use `html('text')`